ccmlib/scylla_docker_cluster.py

```diff
@@ -44,6 +46,15 @@ def _update_config(self):
                 'docker_image': docker_image
             }, f)
 
+    def get_node_ip(self, nodeid):
+        try:
+            node = self.nodelist()[nodeid - 1]
+            return node.address()
+        except IndexError:
+            if nodeid > len(self.nodelist()):
+                nodeid += len(self.nodelist())
+            return self.get_ipformat() % nodeid
```

Comment:
why do we need this fallback ? I don't think we should supply those formatted address, since they aren't going to be true, we'll know the address only once we run the container,
any code that assume to know the address in advance is prone to error, and we should refactor it.